server/src/test/java/org/elasticsearch/action/admin/indices/get/GetIndexResponseTests.java

```diff
@@ -203,4 +204,13 @@ public void testCanOutput622Response() throws IOException {
 
         Assert.assertEquals(TEST_6_3_0_RESPONSE_BYTES, base64OfResponse);
     }
+
+    /**
+     * For xContent roundtrip testing we force the xContent output to still contain types because the parser still expects them.
+     * The new typeless parsing is implemented in the client side GetIndexResponse.
+     */
+    @Override
+    protected ToXContent.Params getToXContentParams() {
+        return new ToXContent.MapParams(Collections.singletonMap(BaseRestHandler.INCLUDE_TYPE_NAME_PARAMETER, "true"));
+    }
```

Comment:
This is forced to "true" now because that is what all the calls from the HLRC using this server side Request/Repsone pair are doing.

Reply:
Related to my question above, can we undo the changes in the server-side `GetIndexResponse#toXContent` so that we always include the type name?

Reply:
See my thinking above.